<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhPreferNumbers</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(177, 177, 177);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButtonMenuCad">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(103, 146, 255);</string>
     </property>
     <property name="text">
      <string>Cadastrar cliente</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMenuPro">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(103, 146, 255);</string>
     </property>
     <property name="text">
      <string>Procurar cliente</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMenuExc">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>350</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(103, 146, 255);</string>
     </property>
     <property name="text">
      <string>Excluir cliente</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMenuMos">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>290</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(103, 146, 255);</string>
     </property>
     <property name="text">
      <string>Mostrar clientes</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="imagem.qrc"/>
 </resources>
 <connections/>
</ui>
